perf_dashboard/templates/index.html

```diff
@@ -96,7 +96,7 @@ <h2>Istio Performance Test Configurations</h2>
                       <li><b>v2-stats-nullvm</b>: Generate Istio standard prometheus metrics using nullvm plugin.</li>
                       <li><b>v2-sd-full-nullvm</b>: Export stackdriver metrics, access logs and edges using nullvm plugin.</li>
                       <li><b>v2-sd-nologging-nullvm</b>: Same as above, but does not export access logs.</li>
-                      <li><b>mixer</b>: Client and server sidecars are present with mixer configured (deprecated since from Istio release 1.7).</li>
+                      <li><b>mixer</b>: Client and server sidecars are present with mixer configured (was deprecated since from Istio release 1.7).</li>
```

Comment:
`since from` does not make sense. Maybe replace with `in`

Reply:
sure